docs/getting-started/installation.md

```diff
@@ -181,6 +181,25 @@ To enable shell autocompletion for uv commands, run one of the following:
 
 Then restart the shell or source the shell config file.
 
+You can also enable shell autocompletion for uvx by running the same commands, replacing `uv` with
```

Comment:
I'm not the author, but my understanding is that `uv generate-shell-completion bash` currently generates the completions for both `uv` and `uvx`. I'm not sure that `uvx generate-shell-completion bash` will work at all.

If my understanding is correct, these docs are misleading, and there isn't really need for docs. The existing instructions will cause completion to work for both `uv` and `uvx`.

It wouldn't be hard to modify this code to generate the completions separately, e.g. something like `uv generate-shell-completion --uvx-only bash`.

Reply:
You’re right, I misread the instructions in the summary. Will revert in the morning (or anyone is welcome to PR).

Reply:
We could expand the docs with the statement that uv and uvx completions are included in the same output.

Not sure if it's worth providing further hints, like if you use a per command file for completions, duplicate the output in both files or symlink uvx -> uv. Could be discussed in future docs changes.